rosidl_typesupport_opensplice_cpp/rosidl_typesupport_opensplice_cpp/rosidl_generator_dds_idl_extension.py

```diff
@@ -30,10 +30,7 @@ def get_include_directives(spec, subfolders):
     # prepend directives to include opensplice specific idl file
     for field in spec.fields:
         if field.type.pkg_name == 'builtin_msgs':
-            includes[:0] = [
-                '#ifndef OSPL_IDL_COMPILER',
-                '#include <dds_dcps.idl>',
-                '#endif']
+            includes.insert(0, '#include <dds_dcps.idl>')
```

Comment:
What's the idea with this change? Why do you no longer need to check if that is defined?